docs/exploration-and-organization/exploration.md

```diff
@@ -98,17 +84,51 @@ Some things to remember with bookmarks:
 - Items that you bookmark will get a boost in your search results (but not the search results of other people).
 - To reorder bookmarks, simply drag and drop them in the sidebar.
 
-## Verified items
+## Search
 
-{% include plans-blockquote.html feature="Verification" %}
+![Search results](./images/search-results.png)
 
-Verified questions and models are marked with a blue checkmark icon:
+Use the search bar to find dashboards, questions, collections, subscriptions, and alerts. You can select from the typeahead's dropdown results, or hit enter to view a search results page. You can also activate the search bar from anywhere by pressing the `/` key.
```

Comment:
we can't search for Models in the search bar?
also do we have a list of shortcuts of using metabase, it would be nice (if we have a lot of them) to have a page about it